great_expectations/rule_based_profiler/rule_based_profiler.py

```diff
@@ -681,7 +700,7 @@ def _reconcile_rule_expectation_configuration_builder_configs(
 
         return list(effective_expectation_configuration_builder_configs.values())
 
-    def _get_rules_as_dict(self) -> Dict[str, Rule]:
+    def get_rules_as_dict(self) -> Dict[str, Rule]:
```

Comment:
Does this need to be public?

Reply:
@donaldheppner Yes -- for the upcoming work on the "Self-Initializing Expectations".  Thanks. P.S.: Please do not merge yet (I am following up on @Shinnnyshinshin 's feedback and will commit an update soon).

Reply:
I have the same question - I'd love to connect on this synchronously if possible.

Reply:
@cdkini -- @donaldheppner and I discussed the need for reconciliation.  Also, I was able to make the `rules` setter "standard" -- please re-review.  Thanks!